build-test-hello-world.yaml

```diff
@@ -2,17 +2,18 @@ version: 1.0-preview-1
 steps:
   # build website and func-test images, concurrently
   - id: build-hello-world
-    build: -t {{.Run.Registry}}/hello-world:{{.Run.ID}} .
+    build: -t {{.Run.Registry}}/hello-world:{{.Run.ID}} -f hello-world.dockerfile .
     when: ["-"]
   - id: build-hello-world-test
-    build: -t hello-world-test .
+    build: -t hello-world-test -f hello-world.dockerfile .
     when: ["-"]
   # run built images to be tested
   - id: hello-world
     cmd: {{.Run.Registry}}/hello-world:{{.Run.ID}}
     when: ["build-hello-world"]
   - id: func-tests
     cmd: hello-world-func-test
```

Comment:
Looks like this `cmd` should either call **hello-world-test**, or the image tag in build-hello-world-test should be changed to **hello-world-func-test**. Getting this:

```console
2018/09/18 15:39:43 Executing step: func-tests
Unable to find image 'hello-world-func-test:latest' locally
docker: Error response from daemon: pull access denied for hello-world-func-test, repository does not exist or may require 'docker login'.
```